app/controllers/ChoiceController.scala

```diff
@@ -59,26 +65,24 @@ class ChoiceController @Inject()(
         (formWithErrors: Form[Choice]) => Future.successful(BadRequest(choicePage(formWithErrors))),
         validChoice => {
           exportsCacheService
-            .update(
-              authenticatedSessionId,
+            .create(
               ExportsDeclaration(
                 None,
                 DeclarationStatus.DRAFT,
-                authenticatedSessionId,
                 createdDateTime = Instant.now,
                 updatedDateTime = Instant.now,
                 validChoice.value
               )
             )
-            .map(_ => {
-              validChoice.value match {
+            .map((created: ExportsDeclaration) => {
```

Comment:
There is a few unnecessary parenthesis, it's difficult to point everything but the last curly brackets in this line and line below normal brackets. It's making code less readable

Reply:
Sorted